content/blog/usemodels-0-0-1/index.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+output: hugodown::hugo_document
+
+slug: usemodels-0-0-1
+title: usemodels 0-0-1
+date: 2020-09-29
+author: Max Kuhn
+description: >
+    The new usemodels R package is a helpful way to automatically generate 
+    tidymodels code. 
+
+photo:
+  url: https://unsplash.com/photos/0TH1H1rq_eY
+  author: Neven Krcmarek
+
+categories: [package] 
+tags: [tidymodels,parsnip,recipes,tune]
+---
+
+```{r startup, include = FALSE}
+library(tidymodels)
+library(usemodels)
+options(width = 90)
+```
+
+We're very excited to announce the first release of the [usemodels](https://usemodels.tidymodels.org/) package. usemodels creates templates for tidymodels analyses so you don't have to write as much new code. 
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("usemodels")
+```
+
+This blog post will show how to use the package. 
+
+Let's start with something simple: creating a glmnet linear regression model for the `mtcars` data using tidymodels. This model is usually tuned over the amount and type of regularization. In tidymodels, there are a few intermediate steps for a glmnet model: 
+
+ * Create a [`parsnip` model object](https://www.tmwr.org/models.html) and define the tuning parameters that we want to optimize. 
+ 
+ * [Create a recipe](https://www.tmwr.org/recipes.html) that, at minimum, centers and scales the predictors. For some data sets, we also need to create dummy variables from any factor-encoded predictor columns. 
+ 
+ * Define a [resampling scheme](https://www.tmwr.org/resampling.html) for our data. 
+ 
+ * Choose a function from the [tune package](https://tune.tidymodels.org/), such as `tune_grid()`, to optimize the parameters. For grid search, we'll also need a grid of candidate parameter values (or let the function choose one for us). 
+ 
+We recognize that this might be more code than you would have had to write compared to a package like `caret`. However, the tidymodels ecosystem enables a wider variety of modeling techniques and has slightly higher complexity.
```

Comment:
```suggestion
We recognize that this might be more code than you would have had to write compared to a package like caret. However, the tidymodels ecosystem enables a wider variety of modeling techniques and is more versatile.
```